pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -418,7 +420,13 @@ func (m *MultishareOpsManager) checkAndStartInstanceDeleteOrShrinkWorkflow(ctx c
 		totalShareCap += share.CapacityBytes
 	}
 	if totalShareCap < instance.CapacityBytes && instance.CapacityBytes > util.MinMultishareInstanceSizeBytes {
-		instance.CapacityBytes = util.Max(totalShareCap, util.MinMultishareInstanceSizeBytes)
+		targetShrinkSizeBytes := util.AlignBytes(totalShareCap, util.GbToBytes(instance.CapacityStepSizeGb))
+		targetShrinkSizeBytes = util.Max(targetShrinkSizeBytes, util.MinMultishareInstanceSizeBytes)
```

Comment:
why do we need this check? isn't the max size and min size all aligned with the step size?

Reply:
say sum of shares size is 500G, and step size is 256G, we want the targetShrinkSizeBytes to be 1TiB (and not 512 GiB)